eraser-map.yaml

```diff
@@ -794,6 +794,16 @@ styles:
         base: points
         texture: pois
         interactive: true
+        
+    icons-overlay:
+        base: points
+        texture: pois
+        interactive: true
+        blend: overlay
```

Comment:
Yes, these icons (for routing and search) should always appear **above** other map icons. Do I set that with an order?